util/lib.py

```
@@ -342,41 +342,54 @@
     if make_base:
-        callstring.extend(['remove'])
+        callstring.extend(["remove"])
         for s in base_iris:
-            callstring.extend(['--base-iri',s])
-        callstring.extend(["--axioms", "external", "--trim", "false", "-p", "false"])
-    
-    ### Measure stuff on base
-    callstring.extend(['measure'])
+            callstring.extend(["--base-iri", s])
+        callstring.extend(
+            ["--axioms", "external", "--trim", "false", "-p", "false"]
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)